tools/c7n_left/c7n_left/cli.py

```diff
@@ -0,0 +1,48 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+#
+import logging
+from pathlib import Path
+
+import click
+from c7n.config import Config
+
+from .output import get_reporter, report_outputs
+from .provider import CollectionRunner
+from .utils import load_policies
+
+
+@click.group()
+def cli():
+    """Shift Left Policy"""
+    logging.basicConfig(level=logging.DEBUG)
+
+
+@cli.command()
+@click.option("--format", default="terraform")
+@click.option("-p", "--policy-dir", type=click.Path())
+@click.option("-d", "--directory", type=click.Path())
+@click.option("-o", "--output", default="cli", type=click.Choice(report_outputs.keys()))
```

Comment:
The multiple output choice is slick and useful, nice. My gut says that `-o` is more commonly short for a destination rather than output format, but that feels like a nit.

Reply:
I think being close to c7n core cli and consistent would be useful at least so you don't have to remember there are differences between the meanings of the args:

```
  -s OUTPUT_DIR, --output-dir OUTPUT_DIR
```

Reply:
Hmm I was wondering about `--format` since it seems sensible enough and has the side benefit of being consistent with `custodian report`. Then I noticed we already have a `--format` option to determine syntax highlighting... but as far as I can tell that CLI option is unused. Looks like `TerraformResource` sets `format = "terraform"` directly - maybe that CLI option isn't necessary?

Reply:
trivy uses --format, checkov uses -o / --output .. I'm game to move to format, the format on terraform resource is more about the lexer for highlighting, which should get renamed if we use format for output type selection.

Reply:
oh.. I guess we're already using format here for the iaac format versus the output format.

Reply:
maybe or --input terraform or --framework to specify in bound source format, there's some auto detection we can do, but its get ambigious for some of the source formats that are doing yaml without some additional content sniffing.

Reply:
Your natural use of "source formats" there suggests maybe `--source-format` or `--source-type` (which might allow a `-t` short form, even if `type` tends to be a pretty overloaded/generic term).

Reply:
thanks, for the discussion and review, It does bring to mind that we should stick a big warning on this cli, not alpha / not compatible guarantees at least for a month or two till we're comfortable on the interface.

Reply:
added the warning on cli ux stability and going to defer this for the moment.